lib-extra/plugins

```diff
@@ -0,0 +1,5 @@
+{erlang_plugins,
+  [
+  {emqx_plugin_template, {git, "https://github.com/z8674558/emqx-plugin-template", {branch, "umbrella"}}}
```

Comment:
can we merge it ?

Reply:
yes. delete CI from emqx-plugin-template.git?

Reply:
merged